packages/reactotron-core-ui/src/timeline-command/timeline-command.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react"
+
+import { TimelineCommandOptions } from "../reactotron-app"
+import { Timestamp } from "./timestamp"
+
+interface Props {
+  command: any // TODO: ...
+  timelineConfig: TimelineCommandOptions
+}
+
+interface State {
+  expanded: boolean
+}
+
+export class TimelineCommand extends React.Component<Props, State> {
+  state = {
+    expanded: false,
+  }
+
+  handleToggle = () => {
+    this.setState(prevState => ({
+      expanded: !prevState.expanded,
+    }))
+  }
+
+  render() {
+    const { command, timelineConfig } = this.props
+    const { expanded } = this.state
+
+    const PreviewComponent = timelineConfig.preview
+    const BodyComponent = timelineConfig.component
+
+    return (
```

Comment:
Need a lot of work here to make it look right and provide all features that were in here before.